src/testRunner/unittests/tsserver/session.ts

```diff
@@ -196,96 +196,97 @@ describe("unittests:: tsserver:: Session:: General functionality", () => {
+            ts.server.protocol.CommandTypes.ImplementationFull,
+            ts.server.protocol.CommandTypes.Exit,
+            ts.server.protocol.CommandTypes.FileReferences,
+            ts.server.protocol.CommandTypes.FileReferencesFull,
+            ts.server.protocol.CommandTypes.Format,
+            ts.server.protocol.CommandTypes.Formatonkey,
+            ts.server.protocol.CommandTypes.FormatFull,
+            ts.server.protocol.CommandTypes.FormatonkeyFull,
+            ts.server.protocol.CommandTypes.FormatRangeFull,
+            ts.server.protocol.CommandTypes.Geterr,
+            ts.server.protocol.CommandTypes.GeterrForProject,
+            ts.server.protocol.CommandTypes.SemanticDiagnosticsSync,
+            ts.server.protocol.CommandTypes.SyntacticDiagnosticsSync,
+            ts.server.protocol.CommandTypes.SuggestionDiagnosticsSync,
+            ts.server.protocol.CommandTypes.NavBar,
+            ts.server.protocol.CommandTypes.NavBarFull,
+            ts.server.protocol.CommandTypes.Navto,
+            ts.server.protocol.CommandTypes.NavtoFull,
+            ts.server.protocol.CommandTypes.NavTree,
+            ts.server.protocol.CommandTypes.NavTreeFull,
+            ts.server.protocol.CommandTypes.Occurrences,
+            ts.server.protocol.CommandTypes.DocumentHighlights,
+            ts.server.protocol.CommandTypes.DocumentHighlightsFull,
+            ts.server.protocol.CommandTypes.JsxClosingTag,
+            ts.server.protocol.CommandTypes.Open,
+            ts.server.protocol.CommandTypes.Quickinfo,
+            ts.server.protocol.CommandTypes.QuickinfoFull,
+            ts.server.protocol.CommandTypes.References,
+            ts.server.protocol.CommandTypes.ReferencesFull,
+            ts.server.protocol.CommandTypes.Reload,
+            ts.server.protocol.CommandTypes.Rename,
+            ts.server.protocol.CommandTypes.RenameInfoFull,
+            ts.server.protocol.CommandTypes.RenameLocationsFull,
+            ts.server.protocol.CommandTypes.Saveto,
+            ts.server.protocol.CommandTypes.SignatureHelp,
+            ts.server.protocol.CommandTypes.SignatureHelpFull,
+            ts.server.protocol.CommandTypes.Status,
+            ts.server.protocol.CommandTypes.TypeDefinition,
+            ts.server.protocol.CommandTypes.ProjectInfo,
+            ts.server.protocol.CommandTypes.ReloadProjects,
+            ts.server.protocol.CommandTypes.Unknown,
+            ts.server.protocol.CommandTypes.OpenExternalProject,
+            ts.server.protocol.CommandTypes.CloseExternalProject,
+            ts.server.protocol.CommandTypes.SynchronizeProjectList,
+            ts.server.protocol.CommandTypes.ApplyChangedToOpenFiles,
+            ts.server.protocol.CommandTypes.EncodedSemanticClassificationsFull,
+            ts.server.protocol.CommandTypes.Cleanup,
+            ts.server.protocol.CommandTypes.GetOutliningSpans,
+            ts.server.protocol.CommandTypes.GetOutliningSpansFull,

```

Comment:
These are the new fixed ones. But, it's weird to me that we encode this huge list rather than just iterating over the enum; seems trivial to do it that way instead and never have this drift or be wrong.

Reply:
Yeah, this "just works":

```ts
const allCommandNames: ts.server.protocol.CommandTypes[] = Object.values((ts.server.protocol as any).CommandTypes);
```

I'm curious if this would be a better idea in the long term.

Reply:
@sheetalkamat Since you're back, do you have any opinion one way or another here? I think the baselines are enough to indicate that something changed, so I think just using `values` here is probably going to be better in the long term to not forget than the comments.

Reply:
Well, the above isn't needed for the beta; I think I'll merge this now and send a followup PR for discussion instead, that way the deprecation is seen.

Reply:
See: https://github.com/microsoft/TypeScript/pull/52319